test/regression/run1/data/parameters.ini

```diff
@@ -1,6 +1,10 @@
 [Settings]
 shb_id=15
 tau=1
+nHealthBoards=15
```

Comment:
You need to add documentation for these new parameters to the `README.md`. The documentation should describe the meaning and use of each one, and should also indicate that these parameters are used purely for validation purposes, rather than in model runs.